blocked-edges/4.12.48-OldKernelHighLoadIOWait.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.12.48
+from: .*
+
+url: https://issues.redhat.com/browse/COS-2705
+name: OldKernelHighLoadIOWait
+message: "After rebooting into kernel-4.18.0-372.88.1.el8_6 or later kernel nodes experience high load average and io_wait times. The nodes might fail to start or stop pods and probes may fail. Workload and host processes may become unresponsive and workload may be disrupted"
```

Comment:
Lets specify the kernel with arch i.e.  4.18.0-372.89.1.el8_6.x86_64

Reply:
I'd rather not pin to an arch without someone explaining why they think this is x86_64-specific.  [OCPBUGS-30096](https://issues.redhat.com/browse/OCPBUGS-30096) mentions x86_64, but that means "at least x86_64 is exposed" and not  necessarly "only x86_64" is exposed.

Reply:
I agree with Trevor. Ignore this suggestion.